bin/preview-deploy/aws.cleanup.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
+
+# Call with the following arguments:
+#    --AWS_REGION <AWS region name>   | The AWS region the instance should be
+#                                     | created in
+#                                     |----------------------------------------
+#    --GH_BOT_PASSWORD <password>     | Username of the Github user to update
+#                                     | comments as. These are the comments
+#                                     | that say the preview deploy was removed
+#                                     |----------------------------------------
+#    --GH_BOT_USER <username>         | Password of the Github user to update
+#                                     | comments as
```

Comment:
Are the username/password descriptors flipped here? `--GH_BOT_PASSWORD` seems like it ought to be described by `Password of the Github user to update comments as`?

Reply:
Oopsie. Yes. I originally had them in a different order, but then I swapped them to be alphabetical and apparently forgot to also swap the descriptions.